app/helpers/layout_helper.rb

```diff
@@ -54,37 +54,24 @@ def popover(title, msg, options = {})
   end
 
   def will_paginate(collection = nil, options = {})
-    options.merge!(:class=>"col-md-7")
-    options[:renderer] ||= "WillPaginate::ActionView::BootstrapLinkRenderer"
-    options[:inner_window] ||= 2
+    options[:renderer] ||= "WillPaginate::ActionView::PatternflyLinkRenderer"
+    options[:page_links] = false
+    options[:container] = false
+    options[:inner_window] ||= 0
     options[:outer_window] ||= 0
-    options[:previous_label] ||= _('&laquo;')
-    options[:next_label] ||= _('&raquo;')
+    options[:previous_label] ||= icon_text("angle-left", "", :kind => "fa")
+    options[:next_label] ||= icon_text("angle-right", "", :kind => "fa")
+    options[:last_label] ||= icon_text("angle-double-right", "", :kind => "fa")
+    options[:first_label] ||= icon_text("angle-double-left", "", :kind => "fa")
```

Comment:
instead of having such a long list of defaults, you can do something like:
```ruby
defaults = {renderer: "WillPaginate::ActionView::PatternflyLinkRenderer", page_links: false, ......}
super(collection, defaults.merge(options))
```